airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_connections.py

```diff
@@ -569,7 +608,7 @@ def test_patch_should_respond_404(self, test_client, body):
 
     @pytest.mark.enable_redact
     @pytest.mark.parametrize(
-        "body, expected_response",
+        "body, expected_response, update_mask",
```

Comment:
I don't think we need the update_mask update.

All fields that are wet will be updated.

Reply:
Oh, we should. Especially because of the Alias and logic you requested to make "better" the current pytest found a regression. Actually needed to scratch my head 3-times until I found the bug.

The more I look on the revised code comparing to the previous list the more I hate it. Over generlized. And error prone because of aliases. So we SHOULD keep this pytest with update mask.